taxi/tests.py

```diff
@@ -2,2 +1,114 @@
+from django.test import TestCase, Client
+from taxi.models import Driver, Manufacturer, Car
+from taxi.forms import DriverCreationForm
+from django.contrib.auth import get_user_model
+from django.urls import reverse
 
-# Create your tests here.
+
+class DriverTestCase(TestCase):
+    def setUp(self):
+        self.manufacturer1 = Manufacturer.objects.create(name="Ivan")
+        self.manufacturer2 = Manufacturer.objects.create(name="Alex")
+        self.car1 = (Car.objects.create
+                     (model="BMW",
+                      manufacturer=self.manufacturer1)
+                     )
+        self.car2 = Car.objects.create(
+            model="Mercedes",
+            manufacturer=self.manufacturer2
+        )
+        self.driver1 = Driver.objects.create(
+            username="driver1",
+            license_number="tes12345"
+        )
+        self.driver2 = Driver.objects.create(
+            username="driver2",
+            license_number="tes23456"
+        )
+
+    def test_car_list(self):
+        self.car1.drivers.set([self.driver1])
+        self.car2.drivers.set([self.driver2])
+        response = self.client.get(CAR_LIST_URL)
+        self.assertEqual(response.status_code, 302)
+
+
+class AdminTestCase(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="password1",
+        )
```

Comment:
It's not a good practice to hardcode sensitive information such as usernames and passwords directly in your test cases. Consider using environment variables or Django's settings for this purpose.